### DISCUSSION TITLE
Request for Detailed and Beginner Friendly Tutorials for Rust Seeking to Develop ABCI Apps

### DISCUSSION THREAD

**Discussion by sikioforie** [informalsystems/tendermint-rs#1063] [Ideas]
Hi, I will like to plead for a detailed beginner friendly tutorials for developer who want to write there own ABCI apps using the library. 

I am a Rust development, I am part of an African dev community of with over 200 developers consisting of 80% Rust Devs. 

We are working to join the Web 3.0 train. I proposed tendermint and most devs in the community are excited why the solution that tendermint provide but are hindered by lack beginner friendly tutorials(in both textual and video) and resources available in our core languages which are Rust.

I will appreciate a feedback, Thank you

**Comment by scalalang2** [comment]
I have same needs.

**Comment by xla** [comment]
You can read up on a project which uses ABCI facilities from this repo to build their app here: https://penumbra.zone/blog/2022-winter-update

**Comment by romac** [comment]
You may also want to take a look at https://github.com/informalsystems/basecoin-rs
